assets/js/util/ModuleCategory.js

```diff
@@ -0,0 +1,16 @@
+import { t } from '@lingui/macro'
+
+const CategoryNames = {
+  Combine: t('js.util.ModuleCategory.CategoryNames.Combine')`Combine`,
+  Scrape: t('js.util.ModuleCategory.CategoryNames.Scrape')`Scrape`,
+  Clean: t('js.util.ModuleCategory.CategoryNames.Clean')`Clean`,
+  Analyze: t('js.util.ModuleCategory.CategoryNames.Analyze')`Analyze`,
+  Visualize: t('js.util.ModuleCategory.CategoryNames.Visualize')`Visualize`,
+  Code: t('js.util.ModuleCategory.CategoryNames.Code')`Code`,
+  'Add data': t('js.util.ModuleCategory.CategoryNames.AddData')`Add data`
+}
```

Comment:
@adamhooper The names of module categories are converted to uppercase when shown as headings in the "Add step" popper, while they are not converted when shown in the success message of importing module from github. Should we ask the translator to define them as uppercase or normal case?

Reply:
For module categories: let's ask them to capitalize the first letter and not the rest. I'll pester Pierre about changing the design to match. Otherwise, we need to ask translators for _both_ ... which is annoying.

Okay, let's capitalize the original text of `rows` and `columns` to `ROWS` and `COLUMNS` and nix the CSS rule.